charts/posthog/values.yaml

```diff
@@ -273,6 +273,10 @@ cert-manager:
   enabled: false
   # -- Whether to install `cert-manager` CRDs.
   installCRDs: true
+  # -- Who to email if the certificate is about to expire
+  # -- Defaults to James.g@posthog.com but feel free (please)
+  # -- to replace this with whoever is managing the stack
+  email: james.g@posthog.com 
```

Comment:
It feels messy for a posthog uses instance to end up triggering an email going to your work email address, this feels like a potential attack vector (not that you are malicious yourself of course! But we shouldn't have access to someone else's communications. I'm sure people will be not setting this.)

Instead can we make this default to empty? What's the behaviour then?

Reply:
100% Updating with something we might be able to use in the future.